tests/infrastructure/certificates.go

```diff
@@ -171,19 +165,12 @@ var _ = DescribeInfra("[rfe_id:4102][crit:medium][vendor:cnv-qe@redhat.com][leve
 
 	DescribeTable("should be rotated when deleted for ", func(secretName string) {
 		By("destroying the certificate")
-		err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-			secret, err := virtClient.CoreV1().Secrets(flags.KubeVirtInstallNamespace).Get(context.Background(), secretName, metav1.GetOptions{})
-			if err != nil {
-				return err
-			}
-			secret.Data = map[string][]byte{
-				"tls.crt": []byte(""),
-				"tls.key": []byte(""),
-			}
-			_, err = virtClient.CoreV1().Secrets(flags.KubeVirtInstallNamespace).Update(context.Background(), secret, metav1.UpdateOptions{})
-
-			return err
-		})
+		secretPatch, err := patch.New(
+			patch.WithReplace("/data/tls.crt", ""),
+			patch.WithReplace("/data/tls.key", ""),
+		).GeneratePayload()
+		Expect(err).ToNot(HaveOccurred())
+		_, err = virtClient.CoreV1().Secrets(flags.KubeVirtInstallNamespace).Patch(context.Background(), components.KubeVirtCASecretName, types.JSONPatchType, secretPatch, metav1.PatchOptions{})
```

Comment:
`components.KubeVirtCASecretName` -> `secretName ?

Reply:
you are correct! fixing